test/controllers/feeds_controller_test.rb

```diff
@@ -26,7 +26,7 @@ def setup
     @f = create_feed published: true
     @f.teams = [@t1, @t2]
     FeedTeam.update_all(shared: true)
-    Bot::Smooch.stubs(:search_for_similar_published_fact_checks).with('text', 'Foo', [@t1.id, @t2.id], nil, @f.id).returns([@pm1, @pm2])
+    Bot::Smooch.stubs(:search_for_similar_published_fact_checks).with('text', 'Foo', [@t1.id, @t2.id], nil, @f.id, skip_cache: false).returns([@pm1, @pm2])
```

Comment:
You can pass a `false` direct without `skip_cache:`

Reply:
Thank you, changes applied!